src/patternfly/components/List/styles.scss

```diff
@@ -0,0 +1,51 @@
+@import "../../patternfly-utilities";
+
+.pf-c-list {
+  // Inline modifier variables
+  --pf-c-list--m-inline--MarginRight: var(--pf-global--spacer--md);
+
+  // Grid modifier variables
+  --pf-c-list--m-grid--GridGap: var(--pf-global--spacer--md);
+
+  // Content variables (no modifier)
+  --pf-c-content--li--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-content--ul--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
+  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
+  --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
+  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
+  --pf-c-content--ul--nested--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-content--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
+
+  &.pf-m-inline {
+    display: flex;
+
```

Comment:
we should `flex-wrap: wrap` yeah? @andresgalante

Reply:
Agreed, will update.